index.js

```diff
@@ -138,8 +181,8 @@ function init (options) {
 
   options = options || {}
 
-  // There is probably 99% chance that the project root directory in located
-  // above the node_modules directory
+  // TODO: Find a better way to locate root directory
+  // The project root directory is probably located above the node_modules directory
```

Comment:
I would rather keep the original copy, because there is a way to set any custom path to package.json like so (from the README):
```js
//
// Import settings from a specific package.json
//
moduleAlias(__dirname + '/package.json')
```